### ISSUE TITLE
Correct image transforms for Yolo-NAS

### ISSUE THREAD

**Issue by VadimShabashov** [Deci-AI/super-gradients#2006]
### 💡 Your Question

It seems there are several places in code where image transforms are defined for Yolo-NAS:
- Predict method on pretrained model:
  ```python
  yolo_nas = super_gradients.training.models.get("yolo_nas_s", pretrained_weights="coco").cuda()
  model_predictions = yolo_nas.predict(
	  "https://deci-pretrained-models.s3.amazonaws.com/sample_images/beatles-abbeyroad.jpg"
  )
  ```
  In this case transforms are defined in `default_yolo_nas_coco_processing_params` function in `processing.py`
  There are `StandardizeImage` and `DetectionCenterPadding`. No RGB to BGR transform.
- Training with `coco_detection_yolo_nas_dataset_params.yaml`.
  Validation transforms include `StandardizeImage`,`DetectionPadToSize` (which is center padding) and RGB to BGR transform.
- Also, there is a popular notebook for fine-tuning Yolo-NAS on custom dataset: https://github.com/roboflow/notebooks/blob/main/notebooks/train-yolo-nas-on-custom-dataset.ipynb
  In this notebook `coco_detection_yolo_format_train` is used for dataset and dataloader creation. Transforms are defined in `coco_detection_yolo_format_base_dataset_params.yaml`. And there is `DetectionPaddedRescale`, which includes bottom right padding. No `StandardizeImage` or RGB to BGR transforms.

So, my question is which transforms should be used for training, validation and prediction with Yolo-NAS?
I want to fine-tune the model with the same transforms that were used during training/validation in order to minimize difference in input data.
